buildconfig/Jenkins/buildscript

```diff
@@ -74,6 +74,17 @@ trap onexit INT TERM EXIT
 ###############################################################################
 export PARAVIEW_DIR=${PARAVIEW_DIR}
 
+###############################################################################
+# Some preprocessing steps on the workspace
+###############################################################################
+BUILD_DIR_REL=build
+BUILD_DIR=$WORKSPACE/$BUILD_DIR_REL
+
+# Clean the source tree to remove stale configured files but make sure to
+# leave external/ and the BUILD_DIR directory intact.
+# There is a later check to see if this is a clean build and remove BUILD_DIR.
+git clean -fdx --exclude=${BUILD_DIR_REL} --exclude=".Xauthority-*"
```

Comment:
Where available, can the full argument names be used for the sake of clarity?

(i.e. `-f` becomes `--force`)

Reply:
:+1:

Reply:
Unfortunately `force` is the only one with a long-form argument. :(